.github/workflows/board.yml

```diff
@@ -0,0 +1,49 @@
+name: Add issues/PRs to project board
+on:
+  workflow_call:
+    inputs:
+      project_id:
+        description: Id of Project in GitHub
+        # To find guid run `gh api -H "Accept: application/vnd.github.inertia-preview+json" orgs/newrelic/projects --jq ".[] | select(.name == \"Node.js Engineering Board\").id"`
+        default: 5864688 # Node.js Engineering Board https://github.com/orgs/newrelic/projects/41
+        required: false
+        type: number
+      todo_column:
+        description: Name of the To-Do column in project
+        default: 'Triage Needed: Unprioritized Features'
+        required: false
+        type: string
+      pr_column:
+        description: Name of the In Review column in project
+        default: 'Needs PR Review'
+        required: false
+        type: string
+    secrets:
+      token:
+        required: true
+
+jobs:
+  assign_pr_to_project:
```

Comment:
since this does issues as well might remove the 'pr' part.

Reply:
good call